### DISCUSSION TITLE
Feature request: Trigger windows volume flyout when using set device volume action

### DISCUSSION THREAD

**Discussion by BuongiornoTexas** [File-New-Project/EarTrumpet#1154] [Ideas]
This is a very minor feature request, but it would be nice if the windows volume flyout could be triggered by a set volume action. This would provide visual feedback to the user that the volume has actually changed. 

The flyout I'm talking about is the small bar that appears on the top left after making a incremental volume change (or a more useful version as provided by the Modern Flyouts app).  

![image](https://user-images.githubusercontent.com/48264358/168700275-32e41a7c-4a80-494b-a482-370c7d38f77a.png)

(I know there is a new flyout coming soon for Windows 11 only - I don't know if this will sort the problem.)

**Comment by tman0004** [comment]
Would be amazing if we get the flyout to appear while we change volume and close after a second or two after no volume change is detected.

**Reply by BuongiornoTexas** [reply]
Just to be sure we are on the same page - the Windows (and Modern) Flyout is already triggered by volume increase and volume decrease events.

This feature request is about showing the flyout when an action changes the volume to a set level (e.g. 20 or 35). I'm keen to see this, as I've spent a lot of time trying to get Windows to not blow my eardrums out when I'm using headphones. As part of this I regularly reset windows to a reference volume (20) - unfortunately, I don't get any feedback that the action has actually triggered. So my current approach is to do a set volume action (no flyout) followed by a volume decrease (flyout shows so I can confirm volume) and then a volume down action to get to the desired target volume. Which is a lot of keystrokes for a simple set volume action and get a visual confirmation of the action.

**Reply by tman0004** [reply]
I should've clarified better, but if I have a keybind set to change a specific app volume, I don't get a flyout. The only flyout I see is the windows default one when i change the entire system volume. Probably a separate feature request now that I think about it.

**Reply by BuongiornoTexas** [reply]
Agreed - that is a slightly different issue - I was focused on the master volume for a device. Does windows even offer a flyout for app volume, or are you talking about showing the ear trumpet flyout briefly?

It'd be nice if they could be joined into a single request, but I suspect implementation may dictate otherwise.

**Reply by tman0004** [reply]
The ear trumpet since like you said don't think windows has an app specific flyout.
